build-ami.sh

```diff
@@ -1,15 +1,22 @@
 #!/bin/bash
-set -ex
+set -exu
 
-APPLIANCE_IMAGE=${1-registry.tld/conjur-appliance}
-APPLIANCE_IMAGE_TAG=${2-latest}
+IMAGE="$1"  # ex: registry.tld/conjur-appliance:4.9-stable
+TAG="${IMAGE##*:}"
 
 if [ ! -f conjur-appliance.tar.gz ]; then
-  docker pull ${APPLIANCE_IMAGE}:${APPLIANCE_IMAGE_TAG}
-  docker save ${APPLIANCE_IMAGE}:${APPLIANCE_IMAGE_TAG} > conjur-appliance.tar
+  docker pull $IMAGE
+  docker save $IMAGE > conjur-appliance.tar
```

Comment:
In the meantime, you should change this so the `docker save` output gets piped into gzip:

```
docker save $IMAGE | gzip > conjur-appliance.tar.gz
```

Not landing the intermediate file on disk will save a fair bit of time.